### ISSUE TITLE
[BUG] Reports do not respect dates and contain errors when using absolute date range

### ISSUE THREAD

**Issue by TonyLovesDevOps** [opensearch-project/dashboards-reports#204]
**Describe the bug**

Generating a report from a dashboard when specifying an absolute date range fails to show the requested date range and the report contains errors showing `Error restoring state from URL
rison decoder error: missing ':'`.

**To Reproduce**
1. Launch a fresh 1.1.0 OpenSearch and OpenSearch Dashboards cluster
2. Load the sample eCommerce data
3. Open the sample `[eCommerce] Revenue Dashboard` dashboard
4. Specify an absolute date from, for example from `October 1, 2021 @ 00:00:00.000` to `now`
5. Click `Reporting` > `Download PDF Report` (also happens w/ PNG reports)

**Expected behavior**
Report shows absolute date range specified in the dashboard and does not contain errors

**OpenSearch Version**
1.1.0 (via `opensearchproject/opensearch:1.1.0` docker image)
```
  "version" : {
    "distribution" : "opensearch",
    "number" : "1.1.0",
    "build_type" : "tar",
    "build_hash" : "15e9f137622d878b79103df8f82d78d782b686a1",
    "build_date" : "2021-10-04T21:29:03.079792Z",
    "build_snapshot" : false,
    "lucene_version" : "8.9.0",
    "minimum_wire_compatibility_version" : "6.8.0",
    "minimum_index_compatibility_version" : "6.0.0-beta1"
  }
```
**Dashboards Version**
1.1.0 (via `opensearchproject/opensearch-dashboards:1.1.0` docker image)

**Plugins**
Defaults -- from `_cat/plugins` API endpoint:
```
opensearch-alerting                  1.1.0.0
opensearch-anomaly-detection         1.1.0.0
opensearch-asynchronous-search       1.1.0.0
opensearch-cross-cluster-replication 1.1.0.0
opensearch-index-management          1.1.0.0
opensearch-job-scheduler             1.1.0.0
opensearch-knn                       1.1.0.0
opensearch-notebooks                 1.1.0.0
opensearch-performance-analyzer      1.1.0.0
opensearch-reports-scheduler         1.1.0.0
opensearch-security                  1.1.0.0
opensearch-sql                       1.1.0.0
```

**Screenshots**

Here's a PNG report of an attempt to export the date range from `October 1, 2021 @ 00:00:00.000` to `now`. Note the errors in the lower right and that the date range shows `Last 7 days` instead of the requested range.

![On_demand_report_with_errors](https://user-images.githubusercontent.com/868644/139486534-637b705e-5760-42a4-8420-8cf8cfb9893e.png)

**Host/Environment (please complete the following information):**
 - OS: Amazon Linux 2 -- run via the [example docker-compose.yaml file](https://opensearch.org/docs/latest/opensearch/install/docker/#sample-docker-compose-file)
 - Browser and version: `Google Chrome Version 95.0.4638.54 (Official Build) (x86_64)`

**Additional context**

This may be fixed by #153 (and potentially just another symptom of issues #61 and/or #88).

**Comment by davidcui1225** [comment]
I tested this with the same eCommerce dashboard and set the absolute date to November 1 and I get the correct time range with no errors in my report. Can you confirm that you still see this issue?

**Comment by TonyLovesDevOps** [comment]
Thanks for taking a look! I just tried again us

> I tested this with the same eCommerce dashboard and set the absolute date to November 1 and I get the correct time range with no errors in my report. Can you confirm that you still see this issue?

Thanks for investigating! I just tried this again using the [example docker-compose.yaml file](https://opensearch.org/docs/latest/opensearch/install/docker/#sample-docker-compose-file) and I can still repro the issue.

**Comment by TonyLovesDevOps** [comment]
Here's a screencast of the repro using the sample data...
![repro](https://user-images.githubusercontent.com/868644/142270725-979551b3-2e15-4d67-bcdf-415e6a5d7499.gif)

**Comment by zhongnansu** [comment]
I am able to reproduce the issue in the 1.1 version, but it seems to be fixed in 1.2. Please use the latest version and try

**Comment by TonyLovesDevOps** [comment]
@zhongnansu thanks for verifying that. I can confirm that the issue is fixed in 1.2.0 running locally.

Do you have any visibility into when 1.2 might be available _on AWS OpenSearch_? I've asked in the [OpenSearch Forum](https://discuss.opendistrocommunity.dev/t/opensearch-roadmap-release-update/6719/9) and on the [AWS forum](https://forums.aws.amazon.com/thread.jspa?threadID=347899&tstart=0) and have heard "we don't know" and 🦗 🦗 , respectively.

**Comment by zhongnansu** [comment]
> @zhongnansu thanks for verifying that. I can confirm that the issue is fixed in 1.2.0 running locally.
> 
> Do you have any visibility into when 1.2 might be available _on AWS OpenSearch_? I've asked in the [OpenSearch Forum](https://discuss.opendistrocommunity.dev/t/opensearch-roadmap-release-update/6719/9) and on the [AWS forum](https://forums.aws.amazon.com/thread.jspa?threadID=347899&tstart=0) and have heard "we don't know" and 🦗 🦗 , respectively.

Sorry, like the answer in the the Opensearch forum says, we don't have a fixed date here for AWS Opensearch release. But I think the long term goal is to have AWS and opensource release at the same day : )
